arrows/ffmpeg/ffmpeg_video_input.cxx

```diff
@@ -131,26 +127,22 @@ class ffmpeg_video_input::priv
   // --------------------------------------------------------------------------
   priv() : m_klv_demuxer( m_klv_timeline ) {}
 
-  // ==================================================================
+  // --------------------------------------------------------------------------
 
-  /*
-   * @brief Whether the video was opened.
-   *
-   * @return \b true if video was opened.
-   */
+  //  @brief Whether the video was opened.
+  //
+  //  @return \b true if video was opened.
```

Comment:
Um... so this is already wrong, but it seems likely this was intended to be `///`, not `// `...

Reply:
Interesting.  It appears the script is replacing `/*` comment blocks with `//` and `/**` blocks with `///`.  I hadn't noticed this before.